### PR TITLE
feat: push events to data-warehouse

### PR DESCRIPTION
**Pull request by marianogoldman**

### CODE REVIEW

File: processor/src/adapters/notifications-service.ts [hunk 1]
```diff
@@ -54,6 +57,13 @@ export async function createNotificationsService(
             try {
               const email = await emailRenderer.renderEmail(subscription.email, notification)
               await sendGridClient.sendEmail(email)
+              await dataWarehouseClient.sendEvent({
```
Comment (subzet) [inline processor/src/adapters/notifications-service.ts:60]: This event wouldn't be necessary as we track this using the sendgrid integration.

File: inbox/src/controllers/handlers/put-subscription-handler.ts [hunk 2]
```diff
@@ -20,6 +20,13 @@ export async function putSubscriptionHandler(
   }
 
   await context.components.db.saveSubscriptionDetails(address, body)
+  await context.components.dataWarehouseClient.sendEvent({
+    context: 'notification_server',
+    event: 'subscription_changed',
+    body: {
+      subscription_details: body
```
Comment (subzet) [inline inbox/src/controllers/handlers/put-subscription-handler.ts:27]: can we add the address of the user in the body?

File: inbox/src/controllers/handlers/unconfirmed-email-handlers.ts [hunk 3]
```diff
@@ -81,6 +91,13 @@ export async function confirmEmailHandler(
 
   await db.saveSubscriptionEmail(address, unconfirmedEmail.email)
   await db.deleteUnconfirmedEmail(address)
+  await dataWarehouseClient.sendEvent({
+    context: 'notification_server',
+    event: 'email_validated',
+    body: {
+      validated_email: unconfirmedEmail.email
```
Comment (subzet) [inline inbox/src/controllers/handlers/unconfirmed-email-handlers.ts:98]: can we add the user's address here as well?

File: inbox/src/controllers/handlers/unconfirmed-email-handlers.ts [hunk 4]
```diff
@@ -43,6 +43,13 @@ export async function storeUnconfirmedEmailHandler(
       }))
     }
     await sendGridClient.sendEmail(email)
+    await dataWarehouseClient.sendEvent({
+      context: 'notification_server',
+      event: 'email_validation_started',
+      body: {
+        email_to_validate: body.email
```
Comment (subzet) [inline inbox/src/controllers/handlers/unconfirmed-email-handlers.ts:50]: can we add the user's address here as well?

File: inbox/src/controllers/handlers/put-subscription-handler.ts (hunk 2 above)
Comment (marianogoldman) [reply inbox/src/controllers/handlers/put-subscription-handler.ts:27]: email address? or eth address?

Comment (subzet) [reply inbox/src/controllers/handlers/put-subscription-handler.ts:27]: eth address sorry

**Pull request merged**